src/kaleidoscope/device/virtual/DefaultHIDReportConsumer.cpp

```diff
@@ -29,7 +29,7 @@
 
 namespace kaleidoscope {
 
-using namespace logging;
+using namespace logging; // NOLINT(build/namespaces)
```

Comment:
Like in a previous case, we could avoid this warning with reasonable ease, at the cost of making the code more verbose than it really needs to be. The cost/benefit ratio of avoiding the `using namespace` construct is - in my opinion - too high to make the fix worth it, and as such, we're better off telling cpplint to ignore it instead.